Scrumdinger/Views/DetailView.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  DetailView.swift
+//  Scrumdinger
+//
+//  Created by Life Boss on 7/16/23.
+//
+
+import SwiftUI
+
+struct DetailView: View {
+    let scrum: DailyScrum
+    
+    @State private var isPresentingEditView = false
+    
+    var body: some View {
+        List {
+            Section (header: Text("Meeting Info")) {
+                NavigationLink (destination: MeetingView()) {
+                    Label("Start Meeting", systemImage: "timer")
+                        .font(.headline)
+                        .foregroundColor(.accentColor)
+                }
+                HStack {
+                    Label("Length", systemImage: "clock")
+                    Spacer()
+                    Text("\(scrum.lengthInMinutes) minutes")
+                }
+                .accessibilityElement(children: .combine)
```

Comment:
☝️ This combines the `Label` and `Text` elements for accessibility users so that they are read as one statement, for example, “Length, 10 minutes.” Without the modifier, VoiceOver users have to swipe again between each element.